.github/compose-lab-releases.json

```diff
@@ -0,0 +1,281 @@
+{
+  "latest": {
+    "id": 157591108,
+    "tag_name": "v2.27.1-desktop.1",
```

Comment:
@sgarcez @silvin-lubecki Seems https://github.com/docker/compose-desktop has not been updated for a while, latest is v2.32.4: https://github.com/docker/compose/releases/tag/v2.32.4

cc @glours